### PR TITLE
API-11684-add-encounter-examples

### PR DESCRIPTION
**Pull request by taylorrassmannlibertyits**
Added new encounter examples.

### CODE REVIEW

File: datamart/dm-records-1008711084V002120/dmEnc2147483643I.json [hunk 1]
```diff
@@ -0,0 +1,46 @@
+{
+  "objectType": "Encounter",
+  "objectVersion": 1,
+  "cdwId": "-2147483643:I",
```
Comment (jblefkove-va) [inline datamart/dm-records-1008711084V002120/dmEnc2147483643I.json:4]: negative?

Comment (taylorrassmannlibertyits) [reply datamart/dm-records-1008711084V002120/dmEnc2147483643I.json:4]: @nmoisan @jtaylor-va @haidong This `cdwId` was negative in the example provided. Was this a mis-type?

File: datamart/dm-records-1008711084V002120/dmEnc2147483643I.json [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+    "display": "WASHINGTON VA MEDICAL CENTER"
+  },
+  "hospitalization": {
+    "dischargeDisposition": {
+      "system": "http://terminology.hl7.org/CodeSystem/discharge-disposition",
+      "code": "X",
+      "display": "RETURN TO COMMUNITY-INDEPENDENT"
+    }
+  },
+  "lastUpdated": "2021-12-01"
```
Comment (jblefkove-va) [inline datamart/dm-records-1008711084V002120/dmEnc2147483643I.json:45]: this needs to be an instant

File: datamart/dm-records-1008711084V002120/dmEnc2147483643I.json (hunk 1 above)
Comment (nmoisan) [reply datamart/dm-records-1008711084V002120/dmEnc2147483643I.json:4]: this is intentional for us, we use negative values to indicate we manufactured the record as opposed to it existing within CDW

File: datamart/dm-records-1008711084V002120/dmEnc2147483643I.json [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+  "objectType": "Encounter",
+  "objectVersion": 1,
+  "cdwId": "2147483643:I",
+  "status": "finished",
+  "class": "IMP",
+  "serviceType": "inpatient",
+  "patient": {
+    "type": "Patient",
+    "reference": "1008711084V002120",
+    "display": "TEST,PATIENT ONE"
```
Comment (jblefkove-va) [inline datamart/dm-records-1008711084V002120/dmEnc2147483643I.json:11]: this is not the display name for this patient

File: datamart/dm-records-1008711084V002120/dmEnc2147483643I.json [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+      "codeVersion": "9",
+      "code": "780.4",
+      "display": "DIZZINESS AND GIDDINESS"
+    }
+  ],
+  "locations": [
+    {
+      "location": {
+        "type": "Location",
+        "reference": "800027766:L",
```
Comment (jblefkove-va) [inline datamart/dm-records-1008711084V002120/dmEnc2147483643I.json:28]: this location does not exist in synthetic data

File: datamart/dm-records-1008711084V002120/dmEnc800022196947O.json [hunk 5]
```diff
@@ -0,0 +1,40 @@
...
+    }
+  ],
+  "period":{
+    "start":"2010-06-08T23:24:02"
+  },
+  "locations":[
+    {
+      "location":{
+        "type":"Location",
+        "reference":"121406:L",
```
Comment (jblefkove-va) [inline datamart/dm-records-1008711084V002120/dmEnc800022196947O.json:29]: ditto

**Review by jblefkove-va — APPROVED** [review]
:shipit:

**Pull request merged**